src/main/java/io/cdap/delta/bigquery/BigQueryEventConsumer.java

```diff
@@ -826,7 +826,7 @@ public static String normalize(String name) {
     }
 
     long baseDelay = Math.min(91, context.getMaxRetrySeconds()) - 1;
-    long maxDelay = Math.max(baseDelay, loadIntervalSeconds);
+    long maxDelay = Math.max(baseDelay, 180);
```

Comment:
thanks for fixing. I think this should be Math.max(baseDelay + 1, loadIntervalSeconds)

Reply:
sure, will apply.